libs/external-db-spanner/src/sql_schema_translator.ts

```diff
@@ -91,12 +91,28 @@ export default class SchemaColumnTranslator implements ISpannerSchemaColumnTrans
                 return `STRING${this.parseLength(2 ** 16)}`
 
             case 'text_large':
+            case 'text_language':
                 return `STRING${this.parseLength(2 ** 32)}`
```

Comment:
`text_language` shouldn't be something like - 'en'?
How did we get this huge size?
it's right to all implementations - I saw you translated language to big texts everywhere, maybe we should reconsider.

Reply:
I don't know how a language item will look like, expect the fact its a string type. 
But you are right I will chnage it to something smaller.